uktrade/data-hub-frontend
# Fix TabNav keyboard interaction

## Description of change

Fixes a bug in nested tab `TabNav` components when focus jumped to the inner `TabNav` when using arrow keys to move the focus of the outer `TabNav`.

## Test instructions

1. Go to the homepage `/` or to the [`TabNav` section of the Storybook](https://uktrade.github.io/data-hub-frontend/?path=/story/tabnav--default).
2. Focus the _tablist_ of the outer `TabNav` component
3. Press the right arrow key repeatedly
4. The focus should cycle through all the tabs in the _tablist_ of the outer `TabNav` and should not jump to the inner `TabNav`
5. Move the focus to the inner `TabNav` by hitting the _tab_ key.
6. Press the right arrow key repeatedly
7. The focus should cycle through all the tabs in the _tablist_ of the inner `TabNav` and should not jump to the outer `TabNav`
8. Repeat steps 4 to 7 with the left arrow key.

## Screenshots

There should be no visual change.
